conf/config-default.yaml

```diff
@@ -296,20 +296,38 @@ nginx_config:                     # Config for render the template to generate n
 #      connect: 2000              # Default 2000ms
 #      send: 2000                 # Default 2000ms
 #      read: 5000                 # Default 5000ms
-#  nacos:                         # Nacos
-#    host:                        # Nacos address(es)
+
+#  nacos:
+#    name: "default"       # Deprecated,see nacos.hosts.name
```

Comment:
The single-register mode that is now supported does not require a name, and I think your defaults in the code can handle that scenario.
So this is not required and should be removed.